supervisor/supervisor.go

```diff
@@ -274,11 +279,17 @@ func (s *Supervisor) Start() error {
 		"memory":      s.machine.Memory,
 		"cpus":        s.machine.Cpus,
 	}).Debug("containerd: supervisor running")
-	go func() {
-		for i := range s.tasks {
-			s.handleTask(i)
-		}
-	}()
+
+	wg := &sync.WaitGroup{}
```

Comment:
Do you mean why this need wg.Add/wg.Done ? In fact, it is unnecessary because the for loop will not exit. This just refers to worker for creating container.